index.js

```diff
@@ -17,13 +17,31 @@
 
 'use strict';
 
-var forEach = require('for-each');
-var warning = require('warning');
-var has = require('has');
-var trim = require('string.prototype.trim');
+/* Should always be false */
+var debug = false;
+
+var forEach = function (iterable, callback) {
+  var keys = Object.keys(iterable);
```

Comment:
this breaks in engines that don't have `Object.keys`, and also exposes the package to breakage from `delete Object.keys`, eg.